src/components/plots/helpers/heatmap/vega/generateVegaData.js

```diff
@@ -14,14 +14,12 @@ const generateVegaData = (
     trackGroupData: [],
   };
 
-  const cells = new Set(cellOrder);
```

Comment:
why not making them a set anymore?

Reply:
I don't see any set-specific actions being carried out over this set, so I think it's not useful to create a new Set for this variable.